src/goto-programs/goto_convert.cpp

```diff
@@ -1974,6 +1974,7 @@ void goto_convert(
 /// \param thread_body: sequence of codet's that can be executed
 ///   in parallel.
 /// \param [out] dest: resulting goto-instructions.
+/// \param mode: language mode.
```

Comment:
`Language mode` (no full stop unless it is a full sentence)

Reply:
I was being consistent with the other parameter documentation for this method.  I find changing style within the documentation for one function is much more off-putting than having a different style between functions.

I thought people would prefer me not to change the documentation for the other parameters to match the coding standard.  Would you prefer me to do that?